bin/Powheg/submitjobs.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
```

Comment:
Hi, do we really need another layer? probably this can stay as private script...